.github/workflows/ci.yaml

```diff
@@ -68,6 +68,8 @@ jobs:
         run: helm repo add stable https://charts.helm.sh/stable && helm repo update
```

Comment:
I think stable can be removed, was only used for KSM

Reply:
Also you need to update `.github/ct.yaml`

Reply:
Indeed!